docs/topics/dokka-module-and-package-docs.md

```diff
@@ -0,0 +1,45 @@
+[//]: # (title: Module documentation)
```

Comment:
I originally had it as `Module and package documentation` but it is very long in the side menu. Hence why I've tried to shorten it. Would `Package documentation` be an option? I liked `Module documentation` because it encompassed any packages within a module.

Reply:
On your second point, I've added this explanation lower in the document.

Reply:
If we have to choose between the two, I think `Module documentation` is indeed better

Let's leave it at that then